src/main/play-doc/operation/Install.md

```diff
@@ -247,15 +247,16 @@ or
 [172.17.0.1]$ sudo yum install conductr-agent-%PLAY_VERSION%-1.noarch.rpm
 ```
 
-then
+Next, you'll need to configure a couple of settings in `conductr-agent.ini` and grant some privileges. ConductR uses [runc](https://runc.io/) to spawn and run OCI-based bundles. This requires the OCI component of ConductR to have root access.
```

Comment:
Elaborate on "why", and also how we've mitigated the risk side given that the root privilege is only granted to the script.